File: C/1142.c
#include <stdio.h>

int main(void) {
  int vezes;
  scanf("%d", &vezes);

  int num = 1;
  int i;
  for(i = 0; i < vezes; i++, num = num + 4){
    printf("%d %d %d PUM\n", num, num+1, num+2);
  }
  
  
  return 0;
}

File: Java/1142.java
import java.io.IOException;
 
/**
 * IMPORTANT: 
 *      O nome da classe deve ser "Main" para que a sua solução execute
 *      Class name must be "Main" for your solution to execute
 *      El nombre de la clase debe ser "Main" para que su solución ejecutar
 */
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int vezes;
        int i, j = 1;
        int aux = 1;
        vezes = sc.nextInt();
        for(i = 1; i <= vezes; i++){
          System.out.println(j + " " + (j+1) + " " + (j+2) +" PUM");
          j = j+4;
        }
    }
    
}